src/io/io_routines.f90

```diff
@@ -593,7 +593,7 @@ module subroutine io_write6d(filename,varname,data_out, dimnames)
             call check( nf90_open(filename, NF90_WRITE, ncid) )
             call check( nf90_redef(ncid) )
         else
-            call check( nf90_create(filename, NF90_CLOBBER, ncid), filename )
+            call check( nf90_create(filename, IOR(NF90_CLOBBER, NF90_HDF5), ncid), filename )
```

Comment:
I did notice that I was having trouble clobbering preload files. I'm guessing this was the change that was causing that.

Reply:
This should still `clobber`, does it not?